jenkins/Jenkinsfile-external

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/groovy
+@Library('h2o3-shared-lib') _
```

Comment:
I though @michal-raska  removed that library, it should not be alive.

Reply:
still works fine, if +@Library('test-shared-lib') _ is on, I will reuse it.

Reply:
The h2o3-shared lib will be removed in a few days, since it looks like the H2O-Pipeline is OK without it. I'll check with @jakubhava before removal

Reply:
`h2o3-shared-lib` is used just for the `cancelPreviousBuilds` method.  I could duplicate it in SW repo, however then I would need to first check-out the SW in order to get this method, but that means that this build would already need to wait for the previous builds. Hence I think that it is a good idea to have this method in the `test-shared-lib`

Reply:
The other solution is to have this `cancelPreviousBuilds` method in each jenkins file, however I would like to avoid that as this method is universal and can be shared amongst different repos as well.